workflows/transcriptomics/alternative-splicing/alternative-splicing.ga

```diff
@@ -0,0 +1,2648 @@
+    "steps": {
+        "0": {
+            "annotation": "Collection of paired-end RNA-seq datasets.",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Collection of paired-end RNA-seq datasets.",
+                    "name": "RNA-seq data collection"
+                }
+            ],
+            "label": "RNA-seq data collection",
+            "name": "Input dataset collection",
+            "outputs": [],
+            "position": {
+                "left": 0,
+                "top": 843
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null, \"collection_type\": \"list:paired\"}",
+            "tool_version": null,
+            "type": "data_collection_input",
+            "uuid": "bcab052d-9a1e-402f-9ce1-81fc88f08908",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "1": {
+            "annotation": "Reference genome annotation in GTF format.",
+            "content_id": null,
+            "errors": null,
+            "id": 1,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Reference genome annotation in GTF format.",
+                    "name": "Genome annotation"
+                }
+            ],
+            "label": "Genome annotation",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 560,
+                "top": 689
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "211faabc-73a3-421c-8baa-081b359af0b3",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "2": {
+            "annotation": "Genome reference in FASTA format.",
+            "content_id": null,
+            "errors": null,
+            "id": 2,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Genome reference in FASTA format.",
+                    "name": "Reference genome"
+                }
+            ],
+            "label": "Reference genome",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 1980,
+                "top": 575
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "adf1c5f8-fa96-40dc-8486-28afa12a0b46",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "3": {
+            "annotation": "IDs of control sequences, corresponding to filenames.",
```

Comment:
I think you need to write a bit more about this. What are these ids, where to get them from etc.